tools/slang-test/slang-test-main.cpp

```diff
@@ -222,7 +222,7 @@ void skipToEndOfLine(char const** ioCursor)
                     cursor++;
                 }
             }
-            ; // fall through to:
+            [[fallthrough]];
```

Comment:
Let's pick a consistent style.

Reply:
I think I've used `[[fallthrough]]` everywhere except for headers compiled by downstream compilers (for which we only require c++14. In those cases I've used `// fall through` which quiets gcc.